README.md

```diff
@@ -53,7 +56,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: uses install-with-cpanm
-        uses: perl-actions/install-with-cpanm@stable
+        uses: perl-actions/install-with-cpanm@v1
```

Comment:
Here my intention was to have double tags as usual, v1 pointing to the last v.1x tag available. You might want to change it back to stable.

Reply:
yeap that works, was going to ask you to preserve `stable` but as long as `v1` points to the latest `v1.x` that works